tests/vmi_networking_test.go

```diff
@@ -892,3 +885,21 @@ func gatewayIPFromCIDR(cidr string) string {
 	ip[oct]++
 	return ip.String()
 }
+
+func isDualStack() bool {
+	provider := os.Getenv("KUBEVIRT_PROVIDER")
```

Comment:
Unfortunately, this will not work because it limits this to running the tests against kubevirtci while we run this against other clusters as well.

This [one](https://github.com/kubevirt/kubevirt/pull/4035/commits/abfb4d07783ee731436e0dc3a2144c4c7fad9357) seem a more  appropriate option although I prefer it through a flag and not an env variable.

For the meantime, I suggest you drop this last commit and instead point to a link on CI that we can see these tests indeed run.
A following PR should add a dedicated test to check dual stack existence, we do not need to check this on every test we run.

Reply:
dropped